conference.js

```diff
@@ -1045,6 +1062,10 @@ export default {
                 (typeof config.hosts.jirecon != "undefined"))?
                 "jirecon" : "colibri";
         }
+        // Initialize in audio only mode
```

Comment:
is this really needed? when you dispatch the audio only mode action we will lest last N to 0, and if there is no channel yet it will be cached.